CHANGELOG

```diff
@@ -4,6 +4,10 @@ django-registration-redux changelog
 
 Version 1.9, TBD
 ----------------
+* BACKWARDS-INCOMPATIBLE CHANGE: Change return signature of
+``RegistrationProfileManager.activate_user``. A tuple of (``User``,
+``activated(bool)``) is now returned.
```

Comment:
It's probably clearer to say "A tuple containing the ``User`` instance and a boolean of whether or not said user was activated".